setup_installer.sh

```diff
@@ -50,14 +62,14 @@ sudo grub-install --root-directory=/tmp/$$.usb --modules="ext2 part_msdos" ${LOO
 cat src/grub.cfg | sed -e "s/@@ARCH@@/${ARCH}/g" -e "s/@@DEB_REL@@/${DEB_REL}/g"| sudo tee /tmp/$$.usb/boot/grub/grub.cfg
 
 sudo mkdir -p /tmp/$$.usb/debian-installer-${DEB_REL}-${ARCH}
-sudo wget -O /tmp/$$.usb/debian-installer-${DEB_REL}-${ARCH}/vmlinuz ${INSTALLER_BASE_URL}/vmlinuz
-sudo wget -O /tmp/$$.usb/debian-installer-${DEB_REL}-${ARCH}/initrd.gz ${INSTALLER_BASE_URL}/initrd.gz
+sudo cp vmlinuz /tmp/$$.usb/debian-installer-${DEB_REL}-${ARCH}/vmlinuz
+sudo cp initrd.gz /tmp/$$.usb/debian-installer-${DEB_REL}-${ARCH}/initrd.gz
 
 sudo cp -r src/preseed /tmp/$$.usb/
 
-sudo mkdir -p /tmp/$$.usb/repo
-sudo cp -v repo/*_all.deb repo/*_${ARCH}.deb /tmp/$$.usb/repo/
-( cd /tmp/$$.usb; dpkg-scanpackages repo/ /dev/null | sudo tee repo/Packages && sudo gzip repo/Packages )
+sudo mkdir -p /tmp/$$.usb/ansible
+sudo cp -va ansible/ /tmp/$$.usb/ansible/ocemr/
+sudo cp -va ocemr.yml /tmp/$$.usb/ansible/
```

Comment:
This file doesn't exist in the repo. If we expect the user to copy it from the other repo please document that and check for the it's missing case before we go through most of the setup script ;-)
    + sudo cp -va ocemr.yml /tmp/11298.usb/ansible/
    cp: cannot stat 'ocemr.yml': No such file or directory

Reply:
fixed